.env

```diff
@@ -3,4 +3,4 @@ POSTGRES_PASS=zammad
 POSTGRES_USER=zammad
 RESTART=always
 # don't forget to add the minus before the version
-VERSION=-4.0.0-25
+VERSION=-4.1.0
```

Comment:
Is there a reason just to use the Zammad version and no package version anymore?

Reply:
Thats just because it was the first version.
https://hub.docker.com/repository/docker/zammad/zammad-docker-compose/tags?page=1&ordering=last_updated
today we have a 4.1.0-1 available :D

Reply:
fixed in: https://github.com/zammad/zammad-docker-compose/pull/215 :D